sdk/batch/Microsoft.Azure.Batch/Tools/ObjectModelCodeGenerator/Spec/PoolSpecification.json

```diff
@@ -266,6 +266,20 @@
             "Type": "IList<Models.MountConfiguration>",
             "Name": "MountConfiguration"
           }
+        },
+        {
+          "Key": {
+            "Type": "Common.NodeCommunicationMode?",
+            "Name": "TargetNodeCommunicationMode",
+            "SummaryComment": "The desired node communication mode for the pool.",
+            "RemarksComment": "If omitted, the default value is default.",
+            "BoundAccess": "read",
```

Comment:
I believe its best to leave the TargetNodeCommunicationNode property as read only for bound objects since according to Xing, only the KeepAlive property under AutoPoolSpecification can be updated in the Batch Service.